.github/CODEOWNERS

```diff
@@ -5,20 +5,44 @@
 
 *       @department-of-veterans-affairs/frontend-review-group
 
-# VSA
-src/applications/proxy-rewrite/redirects/crossDomainRedirects.json @department-of-veterans-affairs/vsa-frontend
-src/applications/find-forms @department-of-veterans-affairs/vsa-frontend
-src/platform/site-wide @department-of-veterans-affairs/vsa-frontend
-src/applications/yellow-ribbon @department-of-veterans-affairs/vsa-frontend
-src/applications/personalization @department-of-veterans-affairs/vsa-frontend
-src/applications/letters @department-of-veterans-affairs/vsa-frontend
-src/applications/disability-benefits @department-of-veterans-affairs/vsa-frontend
-src/platform/user/profile/vet360 @department-of-veterans-affairs/vsa-frontend
+# VSA ----
+
+# Authenticated Experience
+src/applications/personalization @department-of-veterans-affairs/vsa-authd-exp-frontend @department-of-veterans-affairs/vsa-ebenefits-frontend
```

Comment:
so this is saying that `src/applications/personalization` is owned by both `@department-of-veterans-affairs/vsa-authd-exp-frontend` and `@department-of-veterans-affairs/vsa-ebenefits-frontend`?

Reply:
Yep, exactly, but will assigned for review with only one member of either required to approve. In this case, `vsa-ebenefits-frontend` is serving as the backup (although there isn't a distinction of being a "backup")

Reply:
I brought up a need for more developers on a review because some teams only have one FE dev... and many just have two, which doesn't provide enough variety and coverage. At first I was thinking selecting individuals, but @ncksllvn brought up the idea of team-level backups. I think that is a good idea, easier to manage than relying on individual selection (although you could always later go into a GitHub group and start adding individuals to balance out the load). I'm saying "backup" a lot, but it's really more like, "reviewers outside of your team b/c within the team isn't always enough".